test/cli/install/bun-run.test.ts

```diff
@@ -285,6 +285,31 @@ it("should show the correct working directory when run with --cwd", async () =>
   expect(await res.stdout.text()).toMatch(/subdir/);
 });
 
+it("should show the correct working directory when run with --prefix", async () => {
```

Comment:
if feeling fancy, you could also use a `it.each('--cwd', '--prefix')` to ensure its the same code (very minor though and doesn't *really* matter)